client/src/components/Billing/index.js

```diff
@@ -0,0 +1,183 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Button from "@material-ui/core/Button";
+import Card from "@material-ui/core/Card";
+import CardActions from "@material-ui/core/CardActions";
+import CardContent from "@material-ui/core/CardContent";
+import CardHeader from "@material-ui/core/CardHeader";
+import CssBaseline from "@material-ui/core/CssBaseline";
+import Grid from "@material-ui/core/Grid";
+import StarIcon from "@material-ui/icons/StarBorder";
+import Typography from "@material-ui/core/Typography";
+import { withStyles } from "@material-ui/core/styles";
+
+/*
+TODO: 
+1) Change signup link on main navigation to route to here.
+2) Considering deleting signup component if not needed
+3) Make links at bottom of tier choices active to create users
+4) Settle on exact wording to use
+*/
+
+const styles = theme => ({
+  "@global": {
+    body: {
+      backgroundColor: theme.palette.common.white
+    }
+  },
+  layout: {
+    width: "auto",
+    marginLeft: theme.spacing.unit * 3,
+    marginRight: theme.spacing.unit * 3,
+    [theme.breakpoints.up(900 + theme.spacing.unit * 3 * 2)]: {
+      width: 900,
+      marginLeft: "auto",
+      marginRight: "auto"
+    }
+  },
+  heroContent: {
+    maxWidth: 600,
+    margin: "0 auto",
+    padding: `${theme.spacing.unit * 8}px 0 ${theme.spacing.unit * 6}px`
+  },
+  cardHeader: {
+    backgroundColor: theme.palette.grey[200]
+  },
+  cardPricing: {
+    display: "flex",
+    justifyContent: "center",
+    alignItems: "baseline",
+    marginBottom: theme.spacing.unit * 2
+  },
+  cardActions: {
+    [theme.breakpoints.up("sm")]: {
+      paddingBottom: theme.spacing.unit * 2
+    }
+  },
+});
+
+
+/* Filled in wording & pricing for now.  I expect we will decide on 
+exact wording later on */
+
+const tiers = [
+  {
+    title: "Newbie",
+    price: "5",
+    description: ["price indicator", "Help documentation", "Email support"],
+    buttonText: "Sign up for now",
+    buttonVariant: "outlined"
+  },
+  {
+    title: "Investor",
+    subheader: "Most popular",
+    price: "15",
+    description: [
+      "3 indicators included",
+      "Help documentation",
+      "Priority email support"
+    ],
+    buttonText: "Get started",
+    buttonVariant: "contained"
+  },
+  {
+    title: "Pro",
+    price: "30",
+    description: [
+      "5 indicators included",
+      "Help documentation",
+      "Priority email support"
+    ],
+    buttonText: "Join us",
+    buttonVariant: "outlined"
+  }
+];
+
+function Pricing(props) {
```

Comment:
For consistency, could this be `const Pricing = (props) => {...}` instead?

Reply:
We can re-write it latter for now we can merge it. I dont see the issue right this second. Willie has the same in the backend. I cleaned that up just a bit but we may all want to start using a ES6 so it looks cleaner.